projects/atomic-swap-service/headless-wallet.md

```diff
@@ -0,0 +1,118 @@
+# Table of Contents
+- [Summary](#summary)
+- [Acceptance criteria](#acceptance_criteria)
+- [Guide-level explanation](#guide-level-explanation)
+- [Reference-level explanation](#reference-level-explanation)
+- [Rationale and alternatives](#rationale-and-alternatives)
+- [Future possibilities](#future-possibilities)
```

Comment:
I had to activate the `Feature Preview` option on GitHub to see this option, it wasn't available before. It's still unavailable for incognito browsers or such.

Since the main use case for this document is to be read by us on GitHub, I think we don't need it anymore. Removed.